c/examples/testme

```diff
@@ -21,7 +21,7 @@
 # Run the C examples and verify that they behave as expected.
 # Example executables must be in PATH
 
-from test_unittest import unittest
```

Comment:
This file can still be useful for some Python 2.7 vs 3.6 differences, such as assertRegex/assertRegexpMatches as used in dispatch https://github.com/apache/qpid-dispatch/blob/8b91099c6fbbca2e129326e4fcddf5d24a6cd237/tests/system_test.py#L880-L886

I worry that if the file is taken out completely, we may end up adding it back later (with completely different content, of course).

Reply:
We're on the way to removing 2.7 too so I'm not going to worry about this at the moment.